src/Mvc/Mvc.Razor/ref/Microsoft.AspNetCore.Mvc.Razor.netcoreapp3.0.cs

```diff
@@ -290,14 +290,6 @@ public partial class ViewsFeature
         public System.Collections.Generic.IList<Microsoft.AspNetCore.Mvc.Razor.Compilation.CompiledViewDescriptor> ViewDescriptors { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } }
     }
 }
-namespace Microsoft.AspNetCore.Mvc.Razor.Infrastructure
```

Comment:
Since we're dealing w/ partial classes, I suggest (just suggest) instead removing the `public` members from the .Manual file.

Reply:
The reason why I needed to do this is that the public class here has an internal setter on a public property. I don't think we can separate that so I moved the whole type to Manual.cs.